Spring-server/src/main/java/ikpi63holding/postclient/controllers/abstractapi/AbstractDataController.java

```diff
@@ -0,0 +1,51 @@
+package ikpi63holding.postclient.controllers.abstractapi;
+
+import ikpi63holding.postclient.data.maibox.Mailbox;
+import ikpi63holding.postclient.data.user.User;
+import ikpi63holding.postclient.data.user.UserRepository;
+import java.util.List;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.server.ResponseStatusException;
+
+public abstract class AbstractDataController {
```

Comment:
this is a very nice example of OOP-using, we really can make children-classes from this superclass. But I think, we could avoid *abstract*-word 'cause we haven't any abstract methods here